### ISSUE TITLE
Replace template diagram with new version

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/techdocs#218]
Replace the template diagram on [this](https://docs.accordproject.org/) page with the new template diagram and update alt text (see below)
https://docs.accordproject.org/

**OLD**
![image](https://user-images.githubusercontent.com/40569793/67494318-02b46b00-f647-11e9-9eb3-b8027a547c82.png)

**NEW**
![template](https://user-images.githubusercontent.com/40569793/67494341-0a740f80-f647-11e9-9ed6-6b99fb2dc453.png)

Change the alt text to: 

"Model, text, and logic are the three components of a Cicero template."

**Comment by jeromesimeon** [comment]
I think this work is already done? https://docs.accordproject.org/docs/next/accordproject.html

Any place that hasn't been fixed yet?

**Comment by Michael-Grover** [comment]
Here, where the screenshot is from: https://docs.accordproject.org/ 

I should have clarified in the description

**Comment by jeromesimeon** [comment]
> Here, where the screenshot is from: https://docs.accordproject.org/
> 
> I should have clarified in the description

This is the documentation for version `0.13` the new documentation is at the link above. It will be released with `0.20`.

**Comment by UcheSylvester** [comment]
@jeromesimeon, if this issue has not been resolved, then I'd like to take it up :)

**Comment by jeromesimeon** [comment]
@UcheSylvester I still believe it's been resolved. Unless I see evidence otherwise!

**Comment by irmerk** [comment]
I think the old logo still exists in https://docs.accordproject.org/

**Comment by jeromesimeon** [comment]
> I think the old logo still exists in https://docs.accordproject.org/

Good point! The front page is global, so we should switch it once we make `0.20` the official version of the documentation.

**Comment by irmerk** [comment]
Closing this issue for now.
